src/three-components/ARRenderer.ts

```diff
@@ -232,18 +186,20 @@ export class ARRenderer extends EventDispatcher {
   }
 
   [$postSessionCleanup]() {
+    (this.renderer as any).setFramebuffer(null);
+    this.parentRenderer.setRendererSize(1, 1);
```

Comment:
What is this for?

Reply:
I've added a comment, this triggers a resize in the render loop. The parent Renderer code does this also, I was following its example, but I'm not sure if this is the best way to do so.